### ISSUE TITLE
ReadLine: ArgumentOutOfRangeException when going out of screen size

### ISSUE THREAD

**Issue by boriseetgerink** [boriseetgerink/ConsoleExtensions#1]
When entering text longer than the terminal screen width, `Console.CursorLeft` throws an `ArgumentOutOfRangeException`:

```
Unhandled exception. System.ArgumentOutOfRangeException:
The value must be greater than or equal to zero and less than the console's buffer size in that dimension. (Parameter 'left')
Actual value was 121.
   at System.ConsolePal.SetCursorPosition(Int32 left, Int32 top)
   at BorisEetgerink.ConsoleExtensions.ConsoleExtensions.ReadLine(String prompt, String defaultInput)
```

**Comment by boriseetgerink** [comment]
Just extending the `Console.BufferWidth` is not sufficient. `cursorPosition` will probably have to be able to handle multi-line inputs.

**Comment by boriseetgerink** [comment]
I prefer to introduce `PickOne` in 1.2.0 before fixing this in 1.2.1.

**Comment by boriseetgerink** [comment]
When fixing this, also include a ReadLine overload that doesn't require a defaultInput.

**Comment by boriseetgerink** [comment]
Fixed with v1.2.1: https://www.nuget.org/packages/BorisEetgerink.ConsoleExtensions/1.2.1
